### ISSUE TITLE
How to convert the project with "PyTorch"?

### ISSUE THREAD

**Issue by JenniferYingyiWu2020** [reachsumit/deep-unet-for-satellite-image-segmentation#27]
Hi,
    I have seen the deep learning is implemented with "Keras" in your project. However, if I would like to rewrite the model and training using "PyTorch", then how to do converting?
![100](https://user-images.githubusercontent.com/72843232/138410366-af230f2e-fb29-426b-b865-733a321d68cc.png)
![101](https://user-images.githubusercontent.com/72843232/138410378-d5b16e5c-357e-43f6-ba25-33ef2dfe56f9.png)

    So, could you pls give me some suggestions? Thanks!

**Comment by wenwuma** [comment]
你好，
***@***.***，你可以问问他。
祝好




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "reachsumit/deep-unet-for-satellite-image-segmentation"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月22日(星期五) 下午3:17
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[reachsumit/deep-unet-for-satellite-image-segmentation] How to convert the project with "PyTorch"? (Issue #27)





 
Hi,
 I have seen the deep learning is implemented with "Keras" in your project. However, if I would like to rewrite the model and training using "PyTorch", then how to do converting?
 So, could you pls give me some suggestions? Thanks!
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JenniferYingyiWu2020** [comment]
Hi,
    The email address you have mentioned is masked, and I am so confused about it.
***@***.***
    So, could you pls tell me the detailed contact information? Thanks!

**Comment by JenniferYingyiWu2020** [comment]
你好，
    请问你上面的“标有星号*”的邮箱地址具体是怎样的呢？

**Comment by wenwuma** [comment]
你有试过这个地址吗&nbsp;






------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Yingyi ***@***.***&gt;; 
发送时间: 2021年11月22日(星期一) 下午2:28
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [reachsumit/deep-unet-for-satellite-image-segmentation] How to convert the project with &quot;PyTorch&quot;? (Issue #27)





 
你好，
 请问你上面的“标有星号*”的邮箱地址具体是怎样的呢？
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
